scripts/us_census/enhanced_tmcf/process_etmcf.py

```diff
@@ -128,6 +128,7 @@ def _process_var_measured_new_column(
 
     # TODO: remove these checks when a better format for opaque mapping is agreed.
     query_format = query_format.replace("?", "__")
+    query_format = query_format.replace("&", "//")
```

Comment:
maybe `~~` (something not in the URL spec)

Reply:
++ works so trying that